### ISSUE TITLE
documentation goals

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#392]
- [ ] have an examples folder
  - [ ] basic regressor
  - [ ] basic classifier
  - [ ] advanced options
  - [ ] same, but in an ipython notebook
  - [ ] use a large dataset (https://www.kaggle.com/c/porto-seguro-safe-driver-prediction/data)
- [ ] examples to include:
  - [ ] changing model names
  - [ ] feature_scaling
  - [ ] feature_selection
  - [ ] training_params
  - [ ] quantile stuff
  - [ ] nlp
  - [ ] gs_params
  - [ ] categorical_ensembling
  - [ ] feature_learning
  - [ ] comparing multiple models
  - [ ] verbose scoring
  - [ ] X_test
  - [ ] calibrate
  - [ ] lgbm_memory_optimized
  - [ ] ensembling
basically, browse through our test suite, and include a bunch of those examples

**Comment by Sandy4321** [comment]
sounds great, where to read it?
